low-level/sspl_ll_d

```diff
@@ -777,6 +788,13 @@ if __name__ == "__main__":
     signal.signal(signal.SIGUSR2, refresh_iem_log_file)
     #DO NOT EDIT: Marker comment to dynamically add signal handler for SIGUSR1 to generate code coverage report
 
+    # Initialize IEM framework
+    try:
```

Comment:
try except block is also there in `initialize_iem` function. Either use `try/except` block here or in `initialize_iem` function.

Reply:
Ok, then its better to have try-exception block in initialize_iem() as @gargpalak048 had done, but do we need 'raise' again at line 727? As we are handling the exception there already and we dont want SSPL to fail out even after catching that exception
@thavanathan

Reply:
Yeah. we need to remove the `raise` stmt from `initialize_iem` to continue without stopping SSPL. Also there we catched only `EventMessageError`. I believe catching `EventMessageError` is enough to catch initialization failure of IEM framework. Otherwise we need to catch the common `Exception` class instead of `EventMessageError`

Reply:
>  I believe catching EventMessageError is enough to catch initialization failure of IEM framework.

Yes i agree to this

Reply:
Agree. Removed the raise statement